tests/integration/src/test/java/org/mule/test/tck/AssertionMessageProcessorTestCase.java

```diff
@@ -104,13 +113,13 @@ public void processZeroEvents() throws Exception
     @Test
     public void processSomeValidEvents() throws Exception
     {
-        AssertionMessageProcessor asp = new AssertionMessageProcessor();
+        AssertionMessageProcessor asp = createAssertionMessageProcessor();
         asp.setFlowConstruct(flowConstruct);
         asp.setExpression(TRUE_EXPRESSION);
         asp.start();
-        asp.process(mock(MuleEvent.class));
-        asp.process(mock(MuleEvent.class));
-        asp.process(mock(MuleEvent.class));
+        asp.process(mockEvent);
+        asp.process(mockEvent);
+        asp.process(mockEvent);
```

Comment:
Here three different events where processed before your change, now the same event is processed three times, is it ok?